src/reducers/releaseNotesReducer.js

```diff
@@ -27,6 +30,21 @@ export function releaseNotesReducer(state = initialState, action) {
     case PUT_RELEASENOTE_ERROR:
       return doUpdateItem(state, action);
 
+    case FETCH_ALL_RELEASENOTES_PENDING:
+      return update(state, {
+        pending: { $set: true }
+      });
+    case FETCH_ALL_RELEASENOTES_SUCCESS:
+      return update(state, {
+        pending: { $set: false },
+        items: { $set: action.payload },
+        error: { $set: null }
+      });
+    case FETCH_ALL_RELEASENOTES_ERROR:
+      return update(state, {
+        pending: { $set: false },
+        error: { $set: action.payload }
+      });
```

Comment:
He du gjort ditta to plassa? Det samme låg i `ReleaseNoteReducer`

Reply:
Oops, skulle være fikset i #d082aa2